tests/test-project/src/test/editor/extensionEditor.test.ts

```diff
@@ -0,0 +1,140 @@
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import {
+	ActivityBar,
+	EditorView,
+	ExtensionEditorView,
+	ExtensionEditorDetailsSection,
+	ExtensionsViewItem,
+	ExtensionsViewSection,
+	SideBarView,
+	ViewControl,
+	VSBrowser,
+	WebDriver,
+} from 'vscode-extension-tester';
+import * as pjson from '../../../package.json';
+
+describe('Extension Editor', function () {
+	let driver: WebDriver;
+	let viewControl: ViewControl;
+	let extensionsView: SideBarView;
+	let item: ExtensionsViewItem;
+
+	let section: ExtensionsViewSection;
+
+	let extensionEditor: ExtensionEditorView;
+	let extensionEditorDetails: ExtensionEditorDetailsSection;
+
+	before(async function () {
+		driver = VSBrowser.instance.driver;
+		viewControl = (await new ActivityBar().getViewControl('Extensions')) as ViewControl;
+		extensionsView = await viewControl.openView();
+		await driver.wait(async function () {
+			return (await extensionsView.getContent().getSections()).length > 0;
+		});
+
+		const view = await ((await new ActivityBar().getViewControl('Extensions')) as ViewControl).openView();
+
+		await view.getDriver().wait(async function () {
+			return (await view.getContent().getSections()).length > 0;
+		});
+		section = (await view.getContent().getSection('Installed')) as ExtensionsViewSection;
+
+		await section.getDriver().wait(async function () {
+			item = (await section.findItem(`@installed ${pjson.displayName}`)) as ExtensionsViewItem;
+			return item !== undefined;
+		});
+
+		await item.click();
+	});
+
+	after(async function () {
+		await viewControl.closeView();
+		await new EditorView().closeAllEditors();
+	});
+
+	describe('ExtensionEditorView', function () {
+		before(async function () {
+			extensionEditor = new ExtensionEditorView();
+		});
+
+		it('getName', async function () {
+			expect(await extensionEditor.getName()).equal('Test Project');
+		});
+
+		it('getVersion', async function () {
+			expect(await extensionEditor.getVersion()).equal('v0.1.0');
+		});
+
+		it('getPublisher', async function () {
+			expect(await extensionEditor.getPublisher()).equal('ExTester');
+		});
+
+		it('getDescription', async function () {
+			expect(await extensionEditor.getDescription()).equal('Extension dedicated to self-testing the ExTester package');
+		});
+
+		// Not working on test project.
+		it.skip('getCount', async function () {});
+
+		it('getTabs', async function () {
+			const tabs = await extensionEditor.getTabs();
+			expect(tabs).to.have.lengthOf(2);
+			expect(tabs).to.contain('FEATURES');
+		});
+
+		it('switchTabs', async function () {
```

Comment:
_Node: if you decide to change API as I suggested in comments before, just remember to change also in tests_